web/src/types.ts

```diff
@@ -69,3 +69,22 @@ export type UpdateMessageEvent = {
 // TODO/typescript: Move the User and Stream placeholder
 // types to their appropriate modules.
 export type User = Record<string, never>;
+
+// TODO/typescript: Add the QueryWord type  to its appropriate module.
+export type QueryWord = {
+    query_word: string;
+    is_stop_word: boolean;
+};
+// TODO/typescript: Add the SearchData type  to its appropriate module.
+export type SearchData = {
+    query_word: QueryWord[];
+    has_stop_words: boolean;
+    stream_query?: boolean;
+    topic_query?: boolean;
```

Comment:
It should be `?: string` for both `stream_query` and `topic_query`.
https://github.com/zulip/zulip/blob/deabb8ed53f0d8e23a854bf42229bee10af374d9/web/tests/filter.test.js#L503-L504
https://github.com/zulip/zulip/blob/deabb8ed53f0d8e23a854bf42229bee10af374d9/web/src/narrow_banner.js#L38-L39

Reply:
@PIG208 Done. Thanks for your guidance